modules/tinymce/src/plugins/table/test/ts/browser/TabKeyNavigationTest.ts

```diff
@@ -42,4 +44,26 @@ describe('browser.tinymce.plugins.table.TabKeyNavigationTest', () => {
       '</td></tr><tr><td>&nbsp;</td><td>&nbsp;</td></tr></tbody></table><p>x</p>'
     );
   });
+  it('TBA: Fire TableModified event when row are added via Tab key', () => {
+    const editor = hook.editor();
+
+    const events: Array<EditorEvent<TableModifiedEvent>> = [];
+    const logEvent = (event: EditorEvent<TableModifiedEvent>) => {
+      events.push(event);
+    };
+    editor.on('TableModified', logEvent);
+
+    editor.setContent('<table><tbody><tr><td>A1</td><td>A2</td></tr><tr><td>B1</td><td>B2</td></tr></tbody></table><p>x</p>');
+
+    LegacyUnit.setSelection(editor, 'tr:nth-child(2) td:nth-child(2)', 0);
+    assert.lengthOf(events, 0);
+    editor.fire('keydown', { keyCode: Keys.tab() } as KeyboardEvent);
+    TinyAssertions.assertContent(editor,
```

Comment:
I wonder if `assertContentPresence` might be a bit clearer here since we just want to assert a `tr` was added

```
TinyAssertions.assertContentPresence(editor, { 'tr': 3 })
```

Reply:
Yeah, this one is  more expressive.